mat_dp_pipeline/data_sources/tmba.py

```diff
@@ -69,11 +75,13 @@ def __call__(self, output_dir: Path) -> None:
         # TODO: move this. to __init__?
         grouping = ["country", "parameter"]
         for key, targets_frame in targets.groupby(grouping):
-            # TODO: make some assertion that key[0] is a country. In __init__?
-            path = (location_to_path(key[0]),) + key[1:]
+            if key[0] == "NM":
```

Comment:
Also good to add a comment, it isn't obvious.

Reply:
agreed